sdk/keyvault/azure-security-keyvault-secrets/test/ut/secret_client_test.cpp

```diff
@@ -1,15 +1,43 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
+#include "gtest/gtest.h"
 
-#include <gtest/gtest.h>
-
+#include <azure/core/context.hpp>
+#include <azure/identity/client_secret_credential.hpp>
 #include <azure/keyvault/secrets/secret_client.hpp>
```

Comment:
This should be a "" reference
```suggestion
#include "azure/keyvault/secrets/secret_client.hpp"
```